README.md

```diff
@@ -6,83 +6,47 @@
+    </a>
 </p>
 
 ---
 
-**Documentation**: <a href="https://betonquest.github.io/BetonQuest/" target="_blank">https://betonquest.github.io/BetonQuest/</a>
+BetonQuest is an advanced and powerful quest plugin. It offers RPG-style conversations with NPCs and a very flexible quest system.
 
-**Source Code**: <a href="https://github.com/BetonQuest/BetonQuest/" target="_blank">https://github.com/BetonQuest/BetonQuest</a>
+**Advanced Information & Documentation: https://betonquest.github.io/BetonQuest/**
 
 ---
 
-BetonQuest is a Spigot questing plugin. It lets the users create complex quests, NPC conversations and other server mechanics for their players.
-
-This page is contains information important for developers. If you're a user of BetonQuest, head to the [Spigot page](https://www.spigotmc.org/resources/betonquest.2117/) or the [official Wiki](https://github.com/BetonQuest/BetonQuest/wiki). Alternatively, you can ask for support on [Issues](https://github.com/BetonQuest/BetonQuest/issues).
-
-## Getting started
-
-If you're looking for an already compiled binaries visit [Releases](https://github.com/BetonQuest/BetonQuest/releases) for standard releases or [dev website](https://betonquest.pl) for development builds. This section will cover setting up the BetonQuest development workflow.
-
-### Prerequisites
+Join our discord for support: https://discord.com/invite/rK6mfHq 
 
-In order to compile BetonQuest you need to have [JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) installed on your system (version 8 or later). This is basically the Java compiler.
+Report issues here: https://github.com/BetonQuest/BetonQuest/issues/new/choose
 
-In order to easily build the plugin _.jar_ file you'll need [Maven](https://maven.apache.org), the build automation tool. Simply download a binary zip archive and unpack it somewhere on your system. Don't forget to add `bin` directory [to your PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/).
+Developer Documentation here: https://betonquest.github.io/BetonQuest/versions/dev/Developer-Documentation/Info-for-developers/
 
-Additionally, Maven needs the [`JAVA_HOME` environment variable](https://confluence.atlassian.com/doc/setting-the-java_home-variable-in-windows-8895.html) to point to your JDK installation in order to work.
-
-### Building
-
-You can either download the [source code](https://github.com/BetonQuest/BetonQuest/archive/master.zip) directly and unpack it on your system or clone it with Git (recommended):
+---
 
 ```
 git clone https://github.com/BetonQuest/BetonQuest.git
 ```
 
-To compile the plugin in IntelliJ open the command line and navigate to the `BetonQuest-parent` module. In Eclipse, you need to navigate to the `BetonQuest-build` module. Then use Maven to package BetonQuest into a _.jar_ file:
-
+**Make sure to set this git config setting before touching any code:**
 ```
-mvn package
+git config core.autocrlf true
```

Comment:
This is only valid for Windows! Meintion that `git config core.autocrlf input` has to be used for linux.

Reply:
done